code/modules/jobs/job_types/security_officer.dm

```diff
@@ -281,6 +281,13 @@ GLOBAL_LIST_INIT(available_depts, list(SEC_DEPT_ENGINEERING, SEC_DEPT_MEDICAL, S
 	id = /obj/item/card/id/syndicate_command/crew_id
 	backpack_contents = list(/obj/item/storage/box/survival/syndie=1)
 
+/datum/outfit/job/security/pirate/pilot
+	name = "First Mate (pilot)"
+	uniform = /obj/item/clothing/under/costume/singer/yellow
```

Comment:
This seems like an odd choice in uniform for a normal job, considering it's a Vocaloid costume. It also has frankly ugly sprites and isn't likely to get resprited anytime soon.